tests/suites/deploy/deploy_revision.sh

```diff
@@ -11,7 +11,8 @@ run_deploy_revision() {
 	wait_for "juju-qa-test" "$(charm_rev "juju-qa-test" 9)"
 
 	# check resource revision per channel specified.
-	juju resources juju-qa-test --format json | jq -S '.resources[0] | .[ "revision"] == "1"'
```

Comment:
doesn't `jq` return exit code != 0 if `.[ "revision"] != "1"`?

Reply:
No, I tested this -- it returns exit code 0 in both cases, and just prints the JSON result `"true"` or `"false"`. Here's an example (I have my prompt include the last command's exit code):

```
0 ~/w/juju$ echo '{"revision": "4"}' | jq '.["revision"] == "4"'
true
0 ~/w/juju$ echo '{"revision": "4"}' | jq '.["revision"] == "5"'
false
0 ~/w/juju$ 
```

There might be a way to get it to turn the equality comparison to an exist code, I don't know.